### PR TITLE
Separate CR bump and reduced passes

### LINKED ISSUE #401: separate CR bump and reduced passes
Per discussion with @kjozwiak, the test runs for Chromium (minor) bump and the reduced manual passes should be separated as part of the new process.

### PR DESCRIPTION
**Pull request by LaurenWags**
Fix #401

- Created wikitemplate-ReducedDesktop.md
- Created wikitemplate-ReducedAndroid.md
- Updated wikitemplate-minorCRbumpDesktop.md
- Updated wikitemplate-minorCRbumpAndroid.md
- Updated brave_testrun_generator.py to use wikitemplate-ReducedDesktop.md and wikitemplate-ReducedAndroid.md

### CODE REVIEW

File: brave_testrun_generator.py [hunk 1]
```diff
@@ -569,6 +702,132 @@ def android_hf_testruns(milestonever):
...
+
+    print("\nChecklist:")
+    for line in checklist:
+        print(line)
+    print("")
+
+    print("\nAndroid ARM Checklist:")
+    print(android_CRminor_template)
+    print("")
+    AndroidARMtitle = "Manual test run on Android ARM  for " + milestonever
```
Comment (stephendonner) [inline brave_testrun_generator.py:779]: Possible nit: is this extra space intended?  `ARM [two spaces] for `?

File: brave_testrun_generator.py [hunk 2]
```diff
@@ -569,6 +702,132 @@ def android_hf_testruns(milestonever):
...
+                                  body=android_CRminor_template,
+                                  milestone=bc_milestone[milestonever],
+                                  labels=AndroidARMlist)
+
+    print("--------------------------------------------------------\n")
+
+    print("Android Tab Checklist:")
+    print(android_CRminor_template)
+    print("")
+    AndroidTabtitle = "Manual test run on Android Tab  for " + milestonever
```
Comment (stephendonner) [inline brave_testrun_generator.py:797]: Possible nit: is this extra space intended?  `Android Tab [two spaces] for `?

**Review by stephendonner — APPROVED** [review]
++

Note that I defer to @kjozwiak / @srirambv for final r, since I haven't-yet had a chance to set this up via Python and give it a whirl (will do so tomorrow, but didn't want to delay this merge).

**Review by srirambv — APPROVED** [review]
++.

Just the minor nits that @stephendonner pointed. Apart from that looks good. Awesome job on the script update. :+1:

**Review by kjozwiak — CHANGES_REQUESTED** [review]
Awesome job @LaurenWags but looks like there's an issue with not being enable to generate IPFS or Tor passes as per:

image

image

Assuming it's due to adding the two new `Basic` selections which probably offset the last two on the list.

**Comment by srirambv** [comment]
For Tor and IPFS you need to type in the name not the selection number. There is a note added above the selection to type in

**Comment by LaurenWags** [comment]
@kjozwiak I see the same as @srirambv when typing tor or ipfs instead of numbers.

Screen Shot 2022-02-24 at 8 50 33 AM

are the numbers confusing? I could remove them for tor/ipfs if you'd prefer. I kept them since they were there previously, but can change if that's better. Could make the list something like:

```
1. Desktop Release (Full manual pass)
2. Desktop Release (Reduced manual pass for Hotfix)
3. Desktop Release (Basic checks for minor Chromium bump)
4. Android Release (Full manual pass)
5. Android Release (Reduced manual pass for Hotfix)
6. Android Release (Basic checks for minor Chromium bump)
7. iOS Release
Tor
IPFS
```

**Comment by srirambv** [comment]
I think we can leave it as is. Its been like this for a while when the script was redone. also has a note on top as well to indicate it

**Review by kjozwiak — APPROVED** [review]
LGTM 👍 Apologies for the noise! For some reason, I thought we used the numbers to indicate IPFS/Tor but as @srirambv pointed out, you have to specifically type out either `ipfs` or `tor`

**Pull request merged**